src/ImageSharp.Web/Providers/PhysicalFileSystemProvider.cs

```diff
@@ -72,16 +65,16 @@ public bool IsValidRequest(HttpContext context)
         /// <inheritdoc/>
         public Task<IImageResolver> GetAsync(HttpContext context)
         {
-            // Path has already been correctly parsed before here.
-            IFileInfo fileInfo = this.fileProvider.GetFileInfo(context.Request.Path.Value);
+            string path = Path.Join(this.providerRootPath, context.Request.Path.Value);
```

Comment:
I _think_ you're correct here and the path is already normalized since it's specialized `PathString`. [PhysicalFileProvider](https://source.dot.net/#Microsoft.Extensions.FileProviders.Physical/PhysicalFileProvider.cs,228) does a bunch of extra checks because it does not have that guarantee.

Reply:
I've added some path normalization, just to be sure we're not allowing access below the configured provider root path 👍🏻